packages/ethereum/sdk/src/config/utils.ts

```diff
@@ -0,0 +1,73 @@
+import { ZERO_ADDRESS } from "@rarible/types"
+import { merge } from "lodash"
+import type { DeepPick } from "../../../../utils/deep-pick"
+import { id32 } from "../common/id"
+import { FEE_CONFIG_URL } from "./common"
+import type { EthereumConfig } from "./type"
+
+export type EtheriumConfigRequiredFields = DeepPick<
```

Comment:
let's use deep partial type with EthereumConfig as an argument, otherwise we make additional type for EthereumConfig

Reply:
the idea was to highlight that some parts of EthereumConfig are required to override and avoid mistakes when some config parts are not overridden. I removed export for EtheriumConfigRequiredFields to avoid confusion.